src/math/iir_llext/CMakeLists.txt

```diff
@@ -0,0 +1,21 @@
+# Copyright (c) 2024 Intel Corporation.
+# SPDX-License-Identifier: Apache-2.0
+
+if(CONFIG_MATH_IIR_DF1)
+set(df1 ../iir_df1.c ../iir_df1_generic.c ../iir_df1_hifi3.c ../iir_df1_hifi4.c ../iir_df1_hifi5.c)
+else()
+set(df1 "")
+endif()
```

Comment:
No need to change this PR, but please take a look at https://github.com/orgs/thesofproject/discussions/9899 and join the effort (for new files at least).

Reply:
@kv2019i PTL didn't test... So, instead of re-testing, let me fix this and re-push